src/modules/Progress/Progress.js

```diff
@@ -0,0 +1,97 @@
+import React, {Component, PropTypes} from 'react';
+import classNames from 'classnames';
+import $ from 'jquery';
+import META from 'src/utils/Meta';
+
+export default class Progress extends Component {
+  static propTypes = {
+    autoSuccess: PropTypes.bool,
+    children: PropTypes.node,
+    className: PropTypes.string,
+    label: PropTypes.string,
+    limitValues: PropTypes.bool,
+    onActive: PropTypes.func,
+    onChange: PropTypes.func,
+    onError: PropTypes.func,
+    onSuccess: PropTypes.func,
+    onWarning: PropTypes.func,
+    percent: PropTypes.number,
+    precision: PropTypes.number,
+    random: PropTypes.bool,
+    showActivity: PropTypes.bool,
+    total: PropTypes.bool,
+    value: PropTypes.bool,
+  };
+
+  componentDidMount() {
+    this.element = $(this.refs.element);
+    this.element.progress({
```

Comment:
This will return an array of objects with one key/value pair for each prop:
```jsx
[
  {autoSuccess: <this.prop.autoSuccess>},
  {label: <this.prop. label>},
  ...etc
]
```